### PR TITLE
Temporary fix for Capital letters for prompt.py

### PR DESCRIPTION
**Pull request by andering**
Address #265 issue,
but i believe there is more about qtile conversion between keycode and keysyms,
but i will create separate issue for it.

### CODE REVIEW

File: libqtile/widget/prompt.py [hunk 1]
```diff
@@ -616,14 +616,18 @@ def _get_keyhandler(self, k):
                 self.completer.reset()
             return self.keyhandlers[k]
 
-    def handle_KeyPress(self, e):
+    def handle_KeyPress(self, e):
```
Comment (ramnes) [inline libqtile/widget/prompt.py]: whitespace

Comment (flacjacket) [inline libqtile/widget/prompt.py]: Hmm, I'm surprised that doesn't trip our linter, it probably should

Comment (Azd325) [inline libqtile/widget/prompt.py]: https://travis-ci.org/qtile/qtile/jobs/150490065#L223

File: libqtile/widget/prompt.py [hunk 2]
```diff
@@ -25,7 +25,7 @@
 #
 # THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
 # IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVhENT SHALL THE
```
Comment (flacjacket) [inline libqtile/widget/prompt.py]: Not sure how this snuck in there :)

Comment (andering) [inline libqtile/widget/prompt.py]: I'm very sorry, i believe it has to be very annoying for you to correct those things after me.
I'll try be more careful next-time. Fixed. Thank you.

Comment (flacjacket) [inline libqtile/widget/prompt.py]: No worries, we're glad to have people contribute, more developers means more gets done and it's much easier to check for things like this than to do the actual work on the issue.

**Comment by flacjacket** [comment]
Thanks for the fixes, this is in

**Pull request merged**